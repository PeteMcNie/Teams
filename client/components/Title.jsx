import React from 'react'

const Title = () => {
    return (
        <div>
            <h1 className='title'>Teams</h1>
        </div>
    )
}

export default Title