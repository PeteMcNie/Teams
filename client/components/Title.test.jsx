import React from 'react'

import { renderWithRedux } from '../testing/utils'

import Title from './Title'

test('Title component matches snapshot', () => {
  const { container } = renderWithRedux(<Title />)
  expect(container).toMatchSnapshot()
})
