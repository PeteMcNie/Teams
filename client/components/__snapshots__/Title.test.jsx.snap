// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title componet matches snapshot 1`] = `
<div>
  <div>
    <h1
      class="title"
    >
      Teams
    </h1>
  </div>
</div>
`;
